docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -198,13 +198,21 @@ The source name can be set as follows:
 
 * CLI: use the ``--source`` option.
 
-* JDBC: set the ``ApplicationName`` client info property on the ``Connection`` instance.
+* JDBC: set the ``ApplicationName`` client info property on the ``Connection``
+  instance.
+
+* Driver properties: add the ``source`` tag when using a Java application that
```

Comment:
These are the JDBC driver properties to use in applications using the driver. Add this as a third bullet in the list and maybe even link to the JDBC driver properties table.